src/pkg/docs/service/composition/composition.go

```diff
@@ -85,3 +86,33 @@ func (s CompositionService) save(userId string, ehrId string, doc *model.Composi
 
 	return nil
 }
+
+func (s CompositionService) GetCompositionById(userId, ehrId string, uid string) (composition *model.Composition, err error) {
```

Comment:
This function is not needed. Just use func (d *DefaultDocumentService) GetDocIndexByDocId(...)
But when saving a Composition in model.DocumentMeta, DocIdEncrypted must be saved in DocsIndex